Project3/styles.css

```diff
@@ -0,0 +1,111 @@
+input[type="radio"],
+input[type="checkbox"] {
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none;
+    border: 2px solid white;
+    height: 18px;
+    width: 18px;
+    display: inline-block;
+}
+
+/* font-family: 'BenchNine', sans-serif; */
+/* all width 800px 1024px 1120px (form) */
+body {
+    background-color: #163346;
+    align: auto;
+    padding: 16px;
+    margin: auto;
+    max-width: 300px;
+    min-width: 300px;
+    font-family: 'BenchNine', sans-serif;
+}
+
+form {
+    background-color: #1e2019;
+    padding: 8px;
+    max-width: 250px;
+    min-width: 250px
```

Comment:
Same as before; these rules are making it too complicated -- if you want something to be a specific width, just use `width: 250px;`

You use min/max width rules when the main width is set in percentages or some other flexible system, but you want to control its min/max.